test/e2e/common/node/container_probe.go

```diff
@@ -520,7 +520,8 @@ var _ = SIGDescribe("Probing container", func() {
 		Description: A Pod is created with liveness probe on grpc service. Liveness probe on this endpoint will not fail. When liveness probe does not fail then the restart count MUST remain zero.
 	*/
 	ginkgo.It("should *not* be restarted with a GRPC liveness probe [NodeAlphaFeature:GRPCContainerProbe][Feature:GRPCContainerProbe]", func() {
-		e2eskipper.SkipUnlessFeatureGateEnabled(kubefeatures.GRPCContainerProbe)
```

Comment:
I think this should just be removed entirely.

For another alpha feature, ProbeTerminationGracePeriod, because it is tagged `[Feature:...]` it will not be run in e2e suites by default:

https://github.com/kubernetes/kubernetes/blob/806e38aeb7bba6625001c8b1edf92425bd9dc62c/test/e2e/common/node/container_probe.go#L456-L464

It gets pulled into testgrid via https://github.com/kubernetes/test-infra/blob/527290824e5798a90034242949b444f9a7a5468c/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L410

However, it seems something is broken in this job because I'm not seeing most of the alpha features that I'd expect to correspond here, and I am seeing Conformance tests which definitely don't belong in an alpha job: https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-alpha-features

Reply:
replied on Slack. I started looking at this and haven't gotten to the bottom yet. Thus I want to keep this TODO to finish the investigation. I agree with your reasoning.

As per alpha job missing this test, this is yet another thing to look at, maybe later.